docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+license_link: {license_link}                            # Required. if license = other, specify "LICENSE" or "LICENSE.md" to link to a file of that name inside the repo, or a URL to a remote file.
+library_name: {library_name}                            # Optional. Example: keras or any library from https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts.
+tags:
+- {tag_0}                                               # Optional. Example: audio
+- {tag_1}                                               # Optional. Example: automatic-speech-recognition
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+model-index:
+- name: {model_id}                                      # Required. Example: CatClassifier.
+  model: {model_uri}                                    # Required. URI to a repository containing the model file.
+  results:
+  - task:
+      type: {task_type}                                 # Required. Example: image-classification.
+      name: {task_name}                                 # Optional. Example: Image Classification.
+    dataset:
+      type: {dataset_type}                              # Required. Example: common_voice. Link to a repository containing the dataset or a dataset id from https://hf.co/datasets.
+      name: {dataset_name}                              # Required. Example: "Common Voice (French)". A pretty name for the dataset.
+      revision: {dataset_version}                       # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+    metrics:
+    - type: {metric_type}                               # Required. Example: false-positive-rate. Use metric id from https://hf.co/metrics.
+      name: {metric_name}                               # Required. Example: "FPR wrt class 0 restricted to feature gender:0 and age:21".
+      value: {metric_value}                             # Required. Example: 0.75.
+      class: {metric_output_class}                      # Optional. Example: "1". Only relevant for certain metrics such as for example false-positive-rate positive rate.
+      subgroups:
+        - feature: {feature_name}                       # Optional. Example: "gender".
+          group: {feature_subgroup}                     # Optional. Example: "female".
+    measurements:
```

Comment:
What is the difference between a measurement and a metric?

Reply:
Good question. A metric is a performance metric, usually a scalar, things like accuracy, recall, precision, false positive rate, etc. Measurement is a term conjured up by Anne and me to name things like SHAP, Partial Dependence, etc. Those things aren't metrics.

Reply:
It didn't come out of thin air though: https://www.nist.gov/itl/ssd/software-quality-group/metrics-and-measures#Metrics_vs._Measures

Reply:
@ChristopherSpelt could it be that what you have in mind with measurements are related to a model in a different way? What do you think if you read the rest of my comment?

If I see the examples for SHAP and Partial Dependence, I think these are metrics that are informative about the predicted outcome of a machine learning model. If you run these tools in the training phase of a model, the result is a metric so we could store them as a metric.

When we have a model in production and we have predicted outcomes, we also want to be able to log these predicted outcomes in a project/system card. In that section of the card I think using the term `measurement` might be something we can consider.

Reply:
I am not sure if I understand what you are suggesting? The reason why we chose to make a different category called "measurements" is because the nature of some of the fairness tests such as shap, pd, etc are not metrics and cannot be logically captured in that field.

Reply:
Right. That would be yet another thing we can measure. Wouldn't we call something like that `monitoring` or `logging`?

Reply:
We could align on this during our meeting I proposed for the `model_index`. To make it easier to apply the format, I trying to figure out what is the most logical way to define the `measurements`. I try to prevent that we have to create a new minor version of the reporting standard every time we have a new type of measurement we want to include.